### PR TITLE
added: include .m4v in video tag reader

### PR DESCRIPTION
**Pull request by notspiff**
## Description
Forgot this extension..

## Motivation and Context
https://forum.kodi.tv/showthread.php?tid=346981

## How Has This Been Tested?
It has not

### CODE REVIEW

File: xbmc/video/tags/VideoInfoTagLoaderFactory.cpp [hunk 1]
```diff
@@ -38,7 +38,7 @@ IVideoInfoTagLoader* CVideoInfoTagLoaderFactory::CreateLoader(const CFileItem& i
   delete nfo;
 
   if (CServiceBroker::GetSettingsComponent()->GetSettings()->GetBool(CSettings::SETTING_MYVIDEOS_USETAGS) &&
-      (item.IsType(".mkv") || item.IsType(".mp4") || item.IsType(".avi")))
+      (item.IsType(".mkv") || item.IsType(".mp4") || item.IsType(".avi")) || item.isType(".m4v"))
```
Comment (Rechi) [inline xbmc/video/tags/VideoInfoTagLoaderFactory.cpp]:
```suggestion
      (item.IsType(".mkv") || item.IsType(".mp4") || item.IsType(".avi") || item.IsType(".m4v")))
```

Comment (notspiff) [reply xbmc/video/tags/VideoInfoTagLoaderFactory.cpp]: that was stupid..

Comment (Rechi) [reply xbmc/video/tags/VideoInfoTagLoaderFactory.cpp]: looks like you missed the brace change

**Comment by Rechi** [comment]
wrong revision was build
jenkins build this please

**Comment by DaveTBlake** [comment]
This looks more like a fix than an improvement @notspiff , suitable to backport to 18.5 (if we do that)?

**Comment by notspiff** [comment]
sure, if you wanna call it a fix. and yes is can certainly be backported.

**Pull request merged**